packages/lms/src/lib/components/navigation/ProgrammeNav.svelte

```diff
@@ -44,42 +44,20 @@
 		</h2>
```

Comment:
```suggestion
		<h2 class="h2 mb-8">
			{current?.title}
			{current?.type === 'institution' ? 'Programmes' : 'Courses'}
		</h2>
```

It doesn't want me to select the whole h2 for some reason...